extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/ranger/AccessRequest.scala

```diff
@@ -72,4 +74,43 @@ object AccessRequest {
     }
     req
   }
+
+  private def getUserGroups(user: UserGroupInformation, userName: String): util.Set[String] = {
+    val enableOverrideUserGroupFromUserStore = SparkRangerAdminPlugin.getConfig.getBoolean(
```

Comment:
how define this in SparkRangerAdminPlugin like 

```
def authorizeInSingleCall: Boolean = getRangerConf.getBoolean(
    s"ranger.plugin.${getServiceType}.authorize.in.single.call",
    false)


```

Reply:
addressed.